plugins/flytekit-envd/flytekitplugins/envd/image_builder.py

```diff
@@ -68,7 +80,7 @@ def execute_command(command: str):
 
     if p.returncode != 0:
         _, stderr = p.communicate()
-        raise Exception(f"failed to run command {command} with error {stderr}")
+        raise Exception(f"failed to run command {command} with error:\n {stderr.decode()}")
```

Comment:
Change this to raise assertionerror
Or runtime or specific error

Reply:
updated